tests/utils/test_skipna.py

```diff
@@ -0,0 +1,23 @@
+import pandas as pd
+from janitor.utils import skipna
+import numpy as np
+
+import pytest
+
+
+@pytest.mark.functions
+def test_skipna():
+    df = pd.DataFrame({"x": ["a", "b", "c", np.nan], "y": [1, 2, 3, np.nan]})
+    func = lambda s: s + "1"
+
+    # Verify that applying function causes error
+    try:
```

Comment:
(Copy/pasted from other block)

Lines 14-18: These should be caught using a `with pytest.raises` block. You can find an example [here](https://stackoverflow.com/a/56569533/1274908).